pyfar/classes/coordinates.py

```diff
@@ -606,7 +651,261 @@ def csize(self):
         The channel size gives the number of points stored in the coordinates
         object.
         """
-        return self._points.size // 3
+        return self._x.size
+
+    @property
+    def cartesian(self):
```

Comment:
all properties are according to [this figure](https://pyfar.readthedocs.io/en/latest/concepts/pyfar.coordinates.html), does it make sence? phi and theta are usally used as in [wiki](https://en.wikipedia.org/wiki/Spherical_coordinate_system) what should we use?